src/Microsoft.IIS.WebManager/app/runtime/runtime.ts

```diff
@@ -0,0 +1,10 @@
+

```

Comment:
Runtime interface serves as entry point for any future WAC runtime specific logic including detect and install IIS administrator and automatically connect to hosts.

StandardRuntime is invoked for standard entrypoint and these operations would be no-op